website/server/controllers/api-v3/tasks/groups.js

```diff
@@ -13,9 +14,18 @@ import {
   getTasks,
   moveTask,
 } from '../../../libs/taskManager';
+import apiMessages from '../../../libs/apiMessages';
 
 let requiredGroupFields = '_id leader tasksOrder name';
 let types = Tasks.tasksTypes.map(type => `${type}s`);
+
+function canNotEditTasks (group, user, assignedUserId) {
```

Comment:
In general I'd say okay but this PR is adding the same code just written differently in two different places. It won't require much effor to create a function like `user.isSubscribed()` and avoid confusion later